worlds/musedash/docs/setup_es.md

```diff
@@ -17,11 +17,12 @@
 ## Instalar el mod de Archipelago en Muse Dash
 
 1. Descarga [MelonLoader.Installer.exe](https://github.com/LavaGang/MelonLoader/releases/latest) y ejecutalo.
-2. Elije la pestaña "automated", haz clic en el botón "select" y busca tu `MuseDash.exe`. Luego haz clic en "install".
+2. Elije la pestaña "automated", haz clic en el botón "select" y busca tu `MuseDash.exe`.
   - Puedes encontrar la carpeta en Steam buscando el juego en tu biblioteca, haciendo clic derecho sobre el y elegir *Administrar→Ver archivos locales*.
   - Si haces clic en la barra superior que te indica la carpeta en la que estas, te dará la dirección de ésta para que puedas copiarla. Al pegar esa dirección en la ventana que **MelonLoader** abre, irá automaticamente a esa carpeta.
-3. Ejecuta el juego una vez, y espera hasta que aparezca la pantalla de inicio de Muse Dash antes de cerrarlo.
-4. Descarga la última version de [Muse Dash Archipelago Mod](https://github.com/DeamonHunter/ArchipelagoMuseDash/releases/latest) y extraelo en la nueva carpeta creada llamada `/Mods/`, localizada en la carpeta de instalación de Muse Dash.
+3. Desmarque "Latest" y seleccione v0.6.1. Luego haz clic en "install".
```

Comment:
use "Desmarca" instead of "Desmarque" and "selecciona" instead of "seleccione"
current is extremely formal spanish, which doesn't go with the archipelago guide context (also inconsistent with the rest of the guide)
other than that, all good, thanks for taking care of it

Reply:
Applied. Thanks for fixing the google translate part haha.

Reply:
no worries, google never gets context right